build-tools/scripts/MSBuildReferences.projitems

```diff
@@ -20,4 +20,8 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
+  <!-- Copy PackageReference content (as specified above) to OutputDir. This no longer happens by default in short-form projects. -->
+  <PropertyGroup>
+      <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
```

Comment:
Should we put this in `Configuration.props` for all projects at the moment? Less things might break when keeping the old behavior.

Reply:
We could, though I am a bit worried about introducing some more potentially unexpected behavior. I could go either way here.

Reply:
I moved this into `src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj` in https://github.com/xamarin/xamarin-android/pull/4132